internal/controller/factory/pvc.go

```diff
@@ -22,6 +22,6 @@ func GetPVCName(cluster *etcdaenixiov1alpha1.EtcdCluster) string {
 	if len(cluster.Spec.Storage.VolumeClaimTemplate.Name) > 0 {
 		return cluster.Spec.Storage.VolumeClaimTemplate.Name
 	}
-
+	//nolint:goconst
 	return "data"
```

Comment:
What if we create multiple clusters in the same namespace? Will it be handled properly?

Reply:
statefulset prepends it's name to PVC name. For example for `sample` EtcdCluster
```
$ k get pvc
NAME                        STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   VOLUMEATTRIBUTESCLASS   AGE
data-etcdcluster-sample-0   Bound    pvc-d4d05886-4940-43e4-ac54-f9289102ffd8   4Gi        RWO            standard       <unset>                 16h
data-etcdcluster-sample-1   Bound    pvc-e6823320-c2bf-461c-9660-8e29c3bddfe7   4Gi        RWO            standard       <unset>                 13h
data-etcdcluster-sample-2   Bound    pvc-9d4fd6f1-2ead-4088-81f2-dd2e2fc941f1   4Gi        RWO            standard       <unset>                 13h
```